ottreeindex/scripts/README.md

```diff
@@ -3,18 +3,73 @@ This directory contains scripts for setting up the database from the command lin
 # Files:
 
 For command-line setup (for testing not from within pyramid):
+
 * config.yml: YAML file with database settings, including table names.
 * setup_db.py: creates DB tables if do not already exist; optionally deleting them all first
 * load_nexson.py: loads the nexson files into the database and creates the JSON index
-* create_otu_table.py: generates a csv file of otu-tree relationships and then loads this using the COPY command
+* create\_otu\_table.py: generates a csv file of otu-tree relationships and then loads this using the COPY command
+* initializedb.py: the pyramid file for initializing the database
+
+
+# Setting up the database on the command line
+
+## Postgres setup
+
+Do not run the following sequence blindly; it is only a template.
+This was tested on OS X 10.9.
+
+1. Install postgres.  There are other ways to do this.
+
+    `$ brew install postgres`
+
+1. Arrange for postgres to start after reboots.  The .plist file references /usr/local/var/postgres.
+   You will probably  have to change the version number in the path.
+   This step is completely optional; you can start it manually with `postgres -D /usr/local/var/postgres` if you prefer.
+
+    `$ mkdir -p ~/Library/LaunchAgents
+    $ ln -sfv /usr/local/opt/postgresql/*.plist ~/Library/LaunchAgents`
+
+1. Initialize the directory where postgres keeps its databases.
+   (Do this only if the directory does not exist or is empty.)
+
+   `$ initdb /usr/local/var/postgres`
+
+1. Launch postgres DBMS server in background.
+
+    `$ launchctl load ~/Library/LaunchAgents/homebrew.mxcl.postgresql.plist`
 
-Pyramid setup:
-* initializedb.py: the pyramid file for iniatializing the database
```

Comment:
should stay separated from other scripts, but should note that this gets run automatically by pyramid